cmd/report.go

```diff
@@ -71,14 +90,28 @@ provided when the report is generated.
 				return err
 			}
 
-			formatter.PrintCookbooksReportSummary(cookbooksState.Records)
-
+			var outputStr, errorStr, ext string
 			switch cookbooksFlags.format {
 			case "csv":
-				return formatter.StoreCookbooksReportCSV(cookbooksState.Records)
+				outputStr, errorStr = formatter.MakeCookbooksReportCSV(cookbooksState.Records)
+				ext = "csv"
 			default:
-				return formatter.StoreCookbooksReportTXT(cookbooksState.Records)
+				outputStr, errorStr = formatter.MakeCookbooksReportTXT(cookbooksState.Records)
+				ext = "txt"
 			}
+
+			fmt.Print(formatter.MakeCookbooksReportSummary(cookbooksState.Records))
+			fmt.Println()
+			err = saveReport("cookbooks", ext, outputStr)
+			if err != nil {
+				return err
+			}
+			err = saveErrorReport("cookbooks", errorStr)
```

Comment:
now that we are moving things around, can we make these `"cookbooks"` strings constants? same for line `105`
```go
err = saveReport("cookbooks", ext, outputStr)
```